subsys/fb/cfb.c

```diff
@@ -25,11 +25,19 @@ static inline uint8_t byte_reverse(uint8_t b)
 }
 
 struct char_framebuffer {
+#ifdef CONFIG_CHARACTER_FRAMEBUFFER_USE_OFFSCREEN_BUFFER
 	/** Pointer to a buffer in RAM */
 	uint8_t *buf;
 
 	/** Size of the framebuffer */
 	uint32_t size;
+#else
+	/** Temporary buffer for transferring framebuffer */
+	uint8_t *transfer_buf;
+
+	/** Size of temporary buffer */
+	size_t transfer_buf_size;
+#endif
```

Comment:
Please just use buf and size, no need for ifdefery here.

Reply:
Fixed it.